docs/src/references/tedge-config-dir.md

```diff
@@ -0,0 +1,94 @@
+# Custom path for `thin-edge` config files
+
+The `tedge` CLI `--config-dir` option can be used to store the thin-edge
+configuration files in a custom directory.
+
+## Create all the directories
+Create all the directories that are needed by the thin-edge for its operations in a custom path as below.
+
+```shell
+sudo tedge --config-dir /global/path/to/config/dir --init
+```
+
+All the directories will be created in the `/global/path/to/config/dir` directory.
+
+For example:
+
+```shell
+sudo tedge --config-dir /tmp/foo --init
+ls -l /tmp/foo
+total 16
+drwxrwxr-x 2 mosquitto mosquitto 4096 Jun 10 14:49 device-certs
+drwxrwxr-x 2 tedge     tedge     4096 Jun 10 14:49 mosquitto-conf
+drwxrwxr-x 2 tedge     tedge     4096 Jun 10 14:49 operations
+drwxrwxr-x 2 tedge     tedge     4096 Jun 10 14:49 plugins
+```
```

Comment:
This is for me unreadable, putting on the same level a command like `ls -l /tmp/foo`  and an output as `total 16`. 

This is why I prefer to add the `$` prompt. What I see is a screen shot.

Without the prompt, you to invent a way to display file and directory content.

Reply:
Added `$` prompt for all the commands.